source/_integrations/terncy.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: Terncy
+description: Instructions on how to setup Terncy devices within Home Assistant.
+ha_category:
+  - Light
+ha_iot_class: Local Push
+ha_release: '0.60'
```

Comment:
This is a bit old? 😉

Reply:
Since the intergration is still in PR. I'm not sure in which version will it be available for release.
So the 0.60 is more like a place holder to be updated after the integration is merged.

If it's not appropriate, any suggestion for the version number at this stage?